src/libudev/libudev.c

```diff
@@ -173,7 +173,9 @@ _public_ struct udev *udev_new(void) {
                                 if (prio < 0)
                                         log_debug("/etc/udev/udev.conf:%u: invalid log level '%s', ignoring.", line_nr, val);
                                 else
+#ifdef UDEV_PROPAGATE_LOG
                                         log_set_max_level(prio);
+#endif
```

Comment:
well, `man udev_new(3)` says nothing about `/etc/udev/udev.conf`. So, people shouldn't rely on that "feature/bug". Maybe, we should simply add a line to the `NEWS`